### DISCUSSION TITLE
centos8系统下cobbler-web安装不了

### DISCUSSION THREAD

**Discussion by xjr1** [cobbler/cobbler-web#58] [General]
我centos8下，其他cobbler环境都搭建好了，现在cobbler-web不知道怎么安装，按照你们github文档操作，也不会使用

**Comment by SchoolGuy** [comment]
I hope Google Translate did translate this correctly. If it did the answer is the following: This frontend is not ready yet sadly because of the lack of resources currently. I am doing my best but things are moving very slow.

**Comment by SchoolGuy** [comment]
Since there was no response to my answer for a long time I will close this issue.

@xjr1 If you have further questions please use a [GitHub Discussion](https://github.com/cobbler/cobbler-web/discussions), for Feature Requests and Bugs please use the appropriate issue templates.
